lib/ansible/plugins/connection/winrm.py

```diff
@@ -325,6 +331,13 @@ def _reset(self):  # used by win_reboot (and any other action that might need to
         self.shell_id = None
         self._connect()
 
+    def _set_connection_timeout_override(self, timeout):
```

Comment:
I'd like to discuss this with a wider group- rather than requiring per-value override methods, I'd rather have a generic way to set (and possibly query) connection args for situations like this.

Reply:
Do you want me to raise an item on the core meeting agenda? I probably won't be online during the next meeting to talk about it unfortunately.